examples/minimal/packages/contracts/package.json

```diff
@@ -40,7 +40,8 @@
     "prettier-plugin-solidity": "^1.0.0-beta.19",
     "rimraf": "^3.0.2",
     "run-pty": "^4.0.3",
-    "solhint": "^3.3.7",
+    "solhint": "^3.4.1",
+    "solhint-plugin-mud": "file:../../../../packages/solhint-plugin-mud",
```

Comment:
this should prob use `link` rather than `file`?

Reply:
Oh I was thinking I wanted a comment there, but afaik I can't, right?
This is intentional, how pnpm does links conflicts with how solhint requires plugins apparently (my guess is something about flat node_modules, took a while to narrow this down)

Reply:
if we add this to templates, we need to adjust `copy-templates.sh` to also parse `file:` deps (not just `link:`)